Wabbajack.Lib/ABatchProcessor.cs

```diff
@@ -12,7 +13,7 @@ namespace Wabbajack.Lib
 {
     public abstract class ABatchProcessor : IBatchProcessor
     {
-        public WorkQueue Queue { get; private set; }
```

Comment:
This change was inspired because the GUI currently maps/displays the current num cores by watching the WorkQueue observable, which it reaches via this property.  By not being set initially (and not implemented INotifyPropertyChanged, as it's not a GUI object), the late set inside `ConfigureProcessor` was not updating the GUI to be aware that the previously null property was now set.

By being set immediately, and instead just calling `SetActiveThreadsObservable`, it avoids this problem naturally